src/settings/QuickMarcSettings/QuickMarcSettings.js

```diff
@@ -0,0 +1,40 @@
+import { useLocation, useRouteMatch } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+
+import { TitleManager } from '@folio/stripes/core';
+import {
+  CommandList,
+  defaultKeyboardShortcuts,
+} from '@folio/stripes/components';
+import { Settings } from '@folio/stripes/smart-components';
+
+import { MarcBibTemplates } from '../MarcBibTemplates';
+
+const QuickMarcSettings = () => {
+  const match = useRouteMatch();
+  const location = useLocation();
+  const { formatMessage } = useIntl();
+
+  const pages = [
```

Comment:
Let's ask Khalilah or Christine if we can show MARC templates setting now. I think it's not planned for this release yet